src/components/PatentUpload/PatentUpload.css

```diff
@@ -0,0 +1,23 @@
+.protein-container {
+    margin: auto;
+}
+
+.protein-container_input {
```

Comment:
Use __ double underscore to separate block from element`.protein-container__input`